docs/api/catalog-resource.md

```diff
@@ -216,6 +216,7 @@ Name                    | Type                       | Required | Notes
 authors                 | string                     | no       |
 created                 | string                     | no       | A timestamp of when the package was first created. Fallback property: `published`.
 dependencyGroups        | array of objects           | no       | Same format as the [package metadata resource](registration-base-url-resource.md#package-dependency-group)
+deprecation             | object                     | no       | Same format as the [package metadata resource](registration-base-url-resource.md#package-deprecation)
```

Comment:
The comment text is the same as on the line above, but the link is different. That looks suspicious.

Reply:
Please detail which version of registration "adds" this field.

Reply:
@agr - improved the comment on both lines

@loic-sharma - I just discovered that the current implementation puts deprecation information in ALL hives, unlike how the feature was specced. I'm going to investigate whether or not this is a problem and fix the issue if it is.